minetest.conf.example

```diff
@@ -72,6 +72,6 @@ default:torch 99,default:cobble 99
 # Helps rivers create more sound, especially on level sections.
 #river_source_sounds = false
 
-# Enable cloud variation by the 'weather' mod.
+# Enable cloud and shadow intensity variation by the 'weather' mod.
```

Comment:
Well technically it doesn't enable shadow *variation* it enables shadows *at all*

Reply:
Hmm... Fair. I'm not good at wording this.

How about "Enable cloud variation by the 'weather' mod and enable and adjust shadows accordingly" ?

Reply:
+1 for the latter